validator/accounts/v2/new.go

```diff
@@ -130,7 +130,21 @@ func inputKeymanagerKind(cliCtx *cli.Context) (v2keymanager.Kind, error) {
 	return v2keymanager.Kind(selection), nil
 }
 
-func inputNewWalletPassword() (string, error) {
+func inputNewWalletPassword(cliCtx *cli.Context) (string, error) {
+	if cliCtx.IsSet(flags.PasswordFileFlag.Name) {
+		passwordFilePath := cliCtx.String(flags.PasswordFileFlag.Name)
+		data, err := ioutil.ReadFile(passwordFilePath)
+		if err != nil {
+			return "", err
+		}
+		enteredPassword := string(data)
+		fmt.Println(enteredPassword)
```

Comment:
```suggestion
```

Reply:
lol whoops was getting the not strong enough pass error